openhands/runtime/base.py

```diff
@@ -216,56 +239,62 @@ def on_event(self, event: Event) -> None:
         if isinstance(event, Action):
             asyncio.get_event_loop().run_until_complete(self._handle_action(event))
 
+    async def _export_latest_git_provider_tokens(self, event: Action) -> None:
+        """
+        Refresh runtime provider tokens when agent attemps to run action with provider token
+        """
+        if self.user_id:
```

Comment:
```suggestion
        if not self.user_id:
```
Am I sleepy already, or we want the opposite here? 😅

Reply:
HA! I'm the sleepy one